test/integration/smoke/test_vpc_vpn.py

```diff
@@ -468,7 +500,7 @@ def get_ssh_client(self, virtual_machine, services, retries):
 
         return ssh_client
 
-    def create_natrule(self, vpc, vm, public_port, private_port, public_ip, network, services=None):
+    def _create_natrule(self, vpc, vm, public_port, private_port, public_ip, network, services=None):
```

Comment:
a matter of style i guess, i consistently changed private methods/non-tests to be prefixed with an underscore.